logos/src/logos/pipeline/correcting_scheduler.py

```
@@ -134,18 +267,65 @@ def _estimate_ettft(self, model_id: int, provider_id: int, provider_type: str) -
-                    penalty=TIER_THRESHOLDS[ReadinessTier.COLD]["penalty"],
                     reasoning=f"No lanes for logosnode model {model_id}, cold-load required",
+                    state_overhead_s=OVERHEAD_COLD_S,
                 )
-            return estimate_ettft_local(view)
+
+            # Gather infrastructure data for VRAM-aware estimation
+            effective_parallel = 1
+            try:
+                effective_parallel, _ = self._logosnode.get_parallel_capacity(model_id, provider_id)
+            except (KeyError, Exception):
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)